public/js/plugins.js

```diff
@@ -319,5 +319,12 @@ export const officialPlugins = [
     kind: 'menu',
     requireDoc: true,
     position: 'middle'
-  }
+  },
+  {
+    name: 'Sitipe',
+    src: '/src/editors/Sitipe.js',
+    icon: 'developer_board',
```

Comment:
This icon is strongly connected with IED in CoMPAS. Sorry we did not specify this at all. I had a look for another icon and made a proposal. What do you think about this?

Reply:
fixed